src/views/layouts/teamlist.hbs

```diff
@@ -82,7 +83,7 @@
 </div>
 
 <script>
-let sortDirections = [true, true, true, true, true, true, true, true, true, true, true, true, true, true]; // Track sort directions for each column
+let sortDirections = [true, true, true, true, true, true, true, true, true, true, true, true, true, true, true]; // Track sort directions for each column
```

Comment:
is this really the best way to do this?

Reply:
i dont know but it works, It doesn't bother me that much because its on the client side